README.md

```diff
@@ -35,6 +35,7 @@ https://ngrok.zufardhiyaulhaq.com
 ```
 
 ## Values
+see example files [here](examples/config-install/values.yaml)
```

Comment:
I don't think another file is needed, can you change it to `charts/ngrok-operator/values.yaml` instead?

Reply:
also can you update the README in `charts/ngrok-operator/README.md.gotmpl`? thanks

Reply:
yes no need another file, i change to `charts/ngrok-operator/values.yaml` and update `charts/ngrok-operator/README.md.gotmpl`, then is it necesary update charts/ngrok-operator/README.md` too?